[2023-03-29 12:57:06,739] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T12:56:40.158841+00:00 [queued]>
[2023-03-29 12:57:06,791] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T12:56:40.158841+00:00 [queued]>
[2023-03-29 12:57:06,793] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 12:57:06,794] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 12:57:06,796] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 12:57:06,854] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): transform_data_task> on 2023-03-29 12:56:40.158841+00:00
[2023-03-29 12:57:06,865] {standard_task_runner.py:52} INFO - Started process 11542 to run task
[2023-03-29 12:57:06,882] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v2', 'transform_data_task', 'manual__2023-03-29T12:56:40.158841+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpy8s_kt6k', '--error-file', '/tmp/tmppca_047q']
[2023-03-29 12:57:06,886] {standard_task_runner.py:77} INFO - Job 92: Subtask transform_data_task
[2023-03-29 12:57:07,042] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v2.transform_data_task manual__2023-03-29T12:56:40.158841+00:00 [running]> on host 5ebc180c7cc0
[2023-03-29 12:57:07,311] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v2
AIRFLOW_CTX_TASK_ID=transform_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T12:56:40.158841+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T12:56:40.158841+00:00
[2023-03-29 12:58:49,244] {stock_data_transform.py:132} ERROR - An error occurred during transformation: Recursive view `stock_data` detected (cycle: `stock_data` -> `stock_data`)
[2023-03-29 12:58:49,258] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 12:58:49,448] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v2, task_id=transform_data_task, execution_date=20230329T125640, start_date=20230329T125706, end_date=20230329T125849
[2023-03-29 12:58:49,708] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 12:58:50,192] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
